great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -623,9 +623,9 @@ def _build_datetime_columns_rule() -> Rule:
         column_max_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = (
             DataAssistant.commonly_used_parameter_builders.get_column_max_metric_multi_batch_parameter_builder()
         )
-        column_quantile_values_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = (
-            DataAssistant.commonly_used_parameter_builders.get_column_quantile_values_metric_multi_batch_parameter_builder()
-        )
+        # column_quantile_values_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = (
+        #    DataAssistant.commonly_used_parameter_builders.get_column_quantile_values_metric_multi_batch_parameter_builder()
+        # )
```

Comment:
Commenting out until future PR will remove `quantile` metrics for `datetime` columns

Reply:
@Shinnnyshinshin The relevant PR #5862  has been merged.  Thanks.